lib/console.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
 
 const util = require('util');
+const errors = require('internal/errors');
 
 function Console(stdout, stderr) {
   if (!(this instanceof Console)) {
     return new Console(stdout, stderr);
   }
   if (!stdout || typeof stdout.write !== 'function') {
-    throw new TypeError('Console expects a writable stream instance');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'stdout', 'WriteStream');
```

Comment:
Oh right, there are `tty.WriteStream` and `fs.WriteStream`, but then those subclasses `Writable` and the condition here even only checks `.write`, not `instanceof`.